src/main/java/lab/star/firebase/FirebaseMessage/FirebaseMessage.java

```diff
@@ -141,20 +142,18 @@ private ObjectNode getPayload() {
 			object.putPOJO(Constants.JSON_NOTIFICATION, notification);
 		if (data != null)
 			object.putPOJO(Constants.JSON_PAYLOAD, data.getData());
-		if(userId!=null && !userId.isEmpty())
-		object.put(Constants.PARAM_TO, userId);
+		if(regIds!=null && regIds.size()==1)
+		object.put(Constants.PARAM_TO, regIds.iterator().next());
 		object.put(Constants.PARAM_PRIORITY, priority.toString());
 		object.put(Constants.PARAM_TIME_TO_LIVE, ttl);
 		object.put(Constants.PARAM_DELAY_WHILE_IDLE, delayWhileIdeal);
-		if(regIds!=null && !regIds.isEmpty())
+		if(regIds!=null && regIds.size()>1)
```

Comment:
please use this format
if(regIds!=null ){
{
   if (regIds.size()==1){
object.put(Constants.PARAM_TO, regIds.iterator().next());
    }else (regIds.size()>1){
object.putPOJO(Constants.PARAM_REGISTRATION_IDS, regIds);
}

}